src/main/kotlin/com/nextstep/racingcar/domain/Cars.kt

```diff
@@ -9,6 +9,16 @@ class Cars(val cars: List<Car>) {
         }
     }
 
+    fun findWinner(): Cars {
+        val maxPosition = cars.map { it.position }.max()
+
+        if (maxPosition != null) {
+            return Cars(cars.filter { it.isMatchedPosition(maxPosition) })
+        }
+
+        return Cars(listOf())
+    }
```

Comment:
Nullable 변수는 뒤로 끌고가지 말고 가능하다면 앞에서 미리 처리하는게 가독성에 좋습니다.
maxPosition이 null이면 빈 Cars가 반환될테니 아래와 같이 처리하는게 어떨까요?

```suggestion
    fun findWinner(): Cars {
        val maxPosition = cars.map { it.position }.max()
            ?: return Cars(emptyList())

        return Cars(cars.filter { it.isMatchedPosition(maxPosition) })
    }
```